README.md

```diff
@@ -25,22 +24,20 @@ We recommend you install [pre-commit](https://pre-commit.com/) and install the p
 pre-commit install
 ```
 
-If you add new dependencies to the project, put them in the `setup.cfg` file and regenerate the `requirements.txt`
+If you add new dependencies to the project, put them in the `pyproject.toml` file and regenerate the `requirements.txt`
 file using:
 
 ```
-pip-compile --output-file=requirements.txt setup.cfg
+pip-compile --resolver=backtracking --no-emit-index-url pyproject.toml
 ```
 
-Any dependencies for development or testing should go in the appropriate `requirements-(dev|test).in` file, before
-running `pip-compile` against that file.
 
 ## Releasing to PyPI
 
+Releases are handled through GitHub Actions by tagging a revision in GitHub. To release version 0.13.37:
+
 ```
-python -m build
-# Test your upload first
-twine upload -r testpypi dist/*
-# If everything looks good, upload for real
-twine upload dist/*
+# Obviously adjust the version number to the version you're releasing, but it must be of the form vX.Y or vX.Y.Z
```

Comment:
Appreciate the README changes, thanks! Does this replace the existing process as laid out in Confluence? If so, I can update that page to link to the readme once this is merged

Reply:
You wrote it awhile back! [link](https://tackle.atlassian.net/wiki/spaces/IP/pages/781910501/Releasing+Woodchipper+and+other+libraries+to+PyPI )But sounds good, I can keep it current.

Reply:
Lol apologies for the "senior" moment.

Reply:
😅 All good!